src/Extension/Propagator/CloudTrace/_register.php

```diff
@@ -2,11 +2,11 @@
 
 declare(strict_types=1);
 
-use OpenTelemetry\Extension\Propagator\XCloudTrace\XCloudTracePropagator;
+use OpenTelemetry\Extension\Propagator\CloudTrace\CloudTracePropagator;
 use OpenTelemetry\SDK\Common\Configuration\KnownValues;
 use OpenTelemetry\SDK\Registry;
 
 Registry::registerTextMapPropagator(
-    KnownValues::VALUE_XCLOUD_TRACE,
-    XCloudTracePropagator::getInstance()
+    KnownValues::VALUE_CLOUD_TRACE,
```

Comment:
This could be another PR, but if I wanted to configure this via the [OTEL_PROPAGATORS](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.27.0/specification/configuration/sdk-environment-variables.md#general-sdk-configuration) env var, I don't think I can currently choose between "one-way" and "two-way". Maybe this needs to be split into two variables and each registered here?

Reply:
Sorry for the delay, taking a look.